examples/drupal-mysql/README.md

```diff
@@ -0,0 +1,60 @@
+## Drupal / MySQL example
```

Comment:
Shall we create a group json marathon file with the drupal service being a dependency of the mysql one so just one step for deploying?